### DISCUSSION TITLE
How do I find issues I upvoted (or added a reaction)

### DISCUSSION THREAD

**Discussion by kospl** [community/community#11935] [Projects and Issues]
Here is my use case: I remember some time ago I upvoted issue in https://github.com/desktop/desktop/issues that requested multi window support for Desktop application.

When I opened mentioned Issues page, I don't see any available filter for my reactions.

I try search for "involves:@me", but no luck - the issue I'm upvoted is not included in the search results.

I could search by issue key words, but problem is that English is not my native language, and I cannot find the issue by keywords I can provide.

Finally I found it by 3rd synonym search. But here a thing - I already closed the tab, so I should do this all again - to find it. And any next time - in few months when I want to check any updates.

You could say I can subscribe to notifications - but this issue generates huge amount of "me too", "please add it" comments, so I don't think I have to get all those notifications just because once I upvoted the issue.

So please add an option to search for issues I reacted to:

- in a specific repo 
- in any place, at https://github.com/issues - currently I see only "Created, Assigned, Mentioned", possibly to add "Reacted to"/"Upvoted"

**Comment by Pacerier** [comment]
Captured by #60953.

**Comment by adamency** [comment]
Hi @kospl,

Since you made this post, did you come into an alternative solution in the meantime, to find this ?

**Reply by zm-cttae** [reply]
I was given to understand that this was a reverse lookup in the issue database of incredible complexities. Theoretically we're talking exponential expenses, or the creation of a new table that would exist for this specific query.

**Reply by adamency** [reply]
@kospl I understand, thanks for your answer :)
